web/src/actionCreators/index.js

```diff
@@ -36,6 +40,32 @@ const axiSSO = axios.create({
   headers: { Accept: 'application/json', Authorization: authorizationHeader() },
 });
 
+/**
+ * Get user authorization status
+ */
+export const authorizationAction = (id, roles = []) => {
+  return async (dispatch, getState) => {
+    dispatch(authorizationStart());
+
+    try {
+      // TODO: axios does not contain the token in header until refreshing page
+      const res = await axiSSO.get(API.AUTHORIZATION(id), {
+        params: {
+          roles: JSON.stringify(roles),
+        },
+      });
+      const authorizationCode = res.data;
+      return dispatch(authorizationSuccess(authorizationCode));
+    } catch (err) {
+      // authmware in api throws 401 for no role user:
+      if (err.response.status === 401)
```

Comment:
i suggest using curly braces on this `if` statement since it doesn't fit on one line